Assets/Scripts/UI/ItemButton.cs

```diff
@@ -27,5 +59,24 @@ public int Count
                 countLabel.SetText(_count.ToString());
             }
         }
+
+        /// <summary>
+        /// If the button is clicked, the item saved in the _item field is used. Consumables are going to be removed
+        /// from the inventory.
+        /// </summary>
+        public void OnClick() 
+        {
+            if (_item is Consumable)
```

Comment:
```suggestion
            if (_item is Consumable consumable)
```
Dann kannst Du unten gleich consumable.Use() schreiben und musst nicht noch extra casten :) (Und für Equipable auch)